nix/configuration.nix

```diff
@@ -62,11 +62,13 @@
   };
 
   # And configure cage to show fossbeamer on it.
-  services.cage = {
+  services.cage = let
+    defaultConfig = pkgs.writeText "fossbeamer-default-config.json" (builtins.toJSON (import ./default-config.nix));
```

Comment:
Can we just put the config file as literal JSON in the repo (maybe just in the root)? Going that intermediate step in nixlang feels a bit too much.